src/views/Messaging.tsx

```diff
@@ -48,6 +48,7 @@ class Messaging extends Component<MessagingProps, MessagingState> {
     /*
      * Adds message to display in UI
      */
+    // TODO: create message actions
     addMessage = (msg: Types.Message): void => {
```

Comment:
Should we add this before we merge in this PR?

Reply:
Nah, this will probs be a bit more work, and MainHome still has too much in its state for it to be quick change